### DISCUSSION TITLE
The "max_content_width" is not functioning properly.

### DISCUSSION THREAD

**Discussion by xchl** [awcodes/filament-tiptap-editor#210] [Q&A]
https://github.com/awcodes/filament-tiptap-editor/assets/3870041/1964c690-a204-4b00-9324-fe3b8d7a0d5f

`
'max_content_width' => 'full',
`
            
```
TiptapEditor::make('content')
                    ->extraInputAttributes(['style' => 'min-height: 12rem;'])
                                ->profile('default')
                    ->disableFloatingMenus()
                    ->disableBubbleMenus()
                    ->disk('cos') // optional, defaults to config setting
                    ->directory('post') // optional, defaults to config setting
                    ->output(TiptapOutput::Html) // optional, change the format for saved data, default is html
                    ->required()
                    ->columnSpanFull()
```

**Comment by awcodes** [comment]
Seems like you maybe missed a step in setting up the plugin with a custom theme.

https://github.com/awcodes/filament-tiptap-editor/assets/3596800/bb113460-1ac1-476e-a01b-5d8e54a3be1d

**Comment by xchl** [comment]
@awcodes" I'm still facing the same issue even though I started a [new project](https://github.com/xchl/filament-tiptap-editor-for-test/blob/main/app/Filament/Resources/CategoryResource.php)，I've tried multiple times, but it still hasn't worked out.

**Reply by xchl** [reply]
https://github.com/xchl/filament-tiptap-editor-for-test/blob/main/app/Filament/Resources/CategoryResource.php

**Reply by awcodes** [reply]
It is working properly for me. Did you set up your custom theme correctly?

![Screenshot 2023-12-01 at 9 27 46 AM](https://github.com/awcodes/filament-tiptap-editor/assets/3596800/a21cfe21-bf04-49cd-ba64-13ea35a841b1)
![Screenshot 2023-12-01 at 9 28 00 AM](https://github.com/awcodes/filament-tiptap-editor/assets/3596800/aa527d3f-6780-44b7-86f8-1b1903e95e02)

**Reply by xchl** [reply]
Hey Adam, I think I found the problem. I added the CSS "word-break: break-word" and it solved the problem.
`  TiptapEditor::make('content')->extraInputAttributes(['style' => 'word-break: break-word'])
`
